src/components/Containers/Header/Header.js

```diff
@@ -41,6 +41,10 @@ export const Header = () => {
     utils.browser.openInNewTab(DISCORD_LINK_URL, DISCORD_LINK_URL);
   };
 
+  const onTabFaqClick = () => {
+    showFaqMenu();
```

Comment:
If this is all, no need for the `onTabFaqClick` handler. 
Just call `showFaqMenu` directly from the JSX.

Reply:
done